org.osgi.test.junit5/test.bndrun

```diff
@@ -53,4 +53,8 @@
 	org.osgi.test.common;version='[0.9.0,0.9.1)',\
 	org.osgi.test.junit5-tests;version='[0.9.0,0.9.1)',\
 	org.osgi.test.junit5;version='[0.9.0,0.9.1)',\
-	osgi.promise;version='[6.0.0,6.0.1)'
+	osgi.promise;version='[6.0.0,6.0.1)',\
+	junit-jupiter-params;version='[5.6.2,5.6.3)',\
```

Comment:
We have been trying to keep this in sorted order, so we should move the junit things up.